src/posts/paper/14-research-sources.ts

```diff
@@ -16,26 +16,47 @@ Read this section if you are interested in the epistemic nitty-gritty behind our
+which is, to our knowledge, the only study of household tranmission with the Delta variant.
+
+We note that this is only 1.3x the original transmission risk for households members. This could indicate that the original number was too high,
+or it could indicate that households are becoming better at isolating infected individuals, driving down the overall chance of household
+transmission.
+
+Previously we used 30% from the meta-analysis of [Curmei et al.](https://www.medrxiv.org/content/10.1101/2020.05.23.20111559v2) (specifically their updated June 27 version). We explain a bit here about where their estimate comes from.
 
 [Curmei et al.](https://www.medrxiv.org/content/10.1101/2020.05.23.20111559v2) used three different methods to synthesize existing literature. The first method is a re-analysis of nine recent studies, which gives them the central estimate of 30% for the “secondary attack rate” (which is the parameter we care about: the chance of getting infected from a housemate who has COVID). They point out that the estimates vary substantially from source to source, which they conjecture is likely in part due to variations in isolation practices.
 
 Their other two estimation methods use independent datasets. Although these two methods don’t estimate secondary attack rate—rather, they estimate R<sub>h</sub>, the _number_ of within-household transmissions, which is not quite the number we’re looking for—we can nonetheless use this information as corroboration. Specifically, they estimate an R<sub>h </sub>of 0.37 from Vo’, Italy, where average household size was 2.1. If the average infected person infects 0.37 of their 1.1 (excluding self) other household members, that works out to each household member having a 34% risk of being infected. The data suggests this could be substantially larger for older people.
 
 #### Partner
 
-This is a highly speculative number. [Li et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184465/) found a 1.6x higher infection rate among spouses than among other adult household members. This is just one datapoint, but it’s the best we have right now. We use this to adjust the 30% figure from [Curmei et al.](https://www.medrxiv.org/content/10.1101/2020.05.23.20111559v2) up by 1.6x, to get 48% for spouses.
+This is something of a speculative number. We found two studies that compare household transmission among spouses vs non-spouses:
+* [Li et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7184465/) found a 1.6x higher transmission rate among spouses than among other adult household members.
+* [Lewis et al.](https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa1166/5893024) found a 1.4x higher transmission rate.
+
+We use this to adjust the 40% figure from [Allen et al.](https://khub.net/documents/135939561/405676950/Increased+Household+Transmission+of+COVID-19+Cases+-+national+case+study.pdf/7f7764fb-ecb0-da31-77b3-b1a8ef7be9aa) up by 1.5x, to get ~60% for spouses.
```

Comment:
Since this is in the partner section, should this number be 48%?

Reply:
would it help to say "the 40% figure for household members from ..." instead?

and footnote "This number was updated from 48% (which was the previous household transmission number 30% scaled up by 1.6x) to 60% as..."

Reply:
I think its cleaner not to include the part in parenthesis, since its no longer part of the calculation.

The 1.5 here is the average of Li/Lewis multiplied by the new household rate.